packages/user-interface/src/components/OgonePaymentForm.tsx

```diff
@@ -0,0 +1,22 @@
+import { OgonePayment } from "@nl-portal/nl-portal-api";
+import { useEffect, useRef } from "react";
+
+const OgonePaymentForm = ({ formAction, formFields }: OgonePayment) => {
```

Comment:
Zou dit component ook nog gewoon in de hook useOgonePayment geplaatst kunnen worden? of kan dit stukje nog ergens hergebruikt kunnen worden?

Reply:
Wanneer ik dat doe krijg ik deze eslint warning:
` 8:7  warning  Fast refresh only works when a file only exports components. Move your component(s) to a separate file  react-refresh/only-export-components`

Reply:
Ah jammer...